app/src/main/java/com/codesoom/assignment/handler/TaskHttpHandler.java

```diff
@@ -0,0 +1,198 @@
+            }
+        }
+
+        // DELETE /tasks/{id}
+        if (requestMethod.equals(DELETE.name())) {
+            if (!resource.isBlank()) {
+                tasks.remove(getFilteredTask(taskId));
+                handleGet(exchange, content);
+            } else {
+                handleError(exchange, content);
+            }
+        }
+    }
+
+    private void handleGet(HttpExchange exchange, String content) throws IOException {
+        new ResponseSuccess(exchange).sendResponse(content);
+    }
+
+    private void handleGetDetail(HttpExchange exchange, Task filteredTask) throws IOException {
+        new ResponseSuccess(exchange).sendResponse(filteredTask.toString());
+    }
+
+    private void handleCreate(HttpExchange exchange, String body) throws IOException {
+        Task task = contentToTask(body);
+        task.setId(generatedId());
+        tasks.add(task);
+
+        new ResponseCreated(exchange).sendResponse(tasksToJson());
+    }
+
+    private void handleUpdate(HttpExchange exchange, Long taskId, String newTitle) throws IOException {
+        Task filteredTask = getFilteredTask(taskId);
+        if (filteredTask == null) {
+            new ResponseNotFound(exchange);
+        }
+
+        filteredTask.setTitle(newTitle);
+
+        new ResponseSuccess(exchange).sendResponse(tasksToJson());
+    }
+
+    private void handleError(HttpExchange exchange, String content) throws IOException {
+        new ResponseNotFound(exchange).sendResponse(content);
+    }
+
+    /**
+     * value값이 int형인지 확인
+     * @param value
+     * @return true || false
+     */
+    private boolean isNumeric(String value) {
+        return value != null && value.matches("[0-9.]+");
+    }
+
+    /**
+     * 넘어온 리소스 값으로 Task 필터링하여 반환
+     *
+     * @param taskId
+     * @return 필터링된 Task
+     * @throws IOException
+     */
+    private Task getFilteredTask(Long taskId) {
+        //TODO stream filter 찾아보기
+       return tasks.stream()
+               .filter(t -> t.getId().equals(taskId))
+               .findFirst()
+               .orElse(null);
+    }
+
+    /**
+     * requestBody에서 받아온 content를 Task 자바 객체로 변환
+     * @param content
+     * @return Task 객체
+     * @throws JsonProcessingException
+     */
+    private Task contentToTask(String content) throws JsonProcessingException {
+        return objectMapper.readValue(content, Task.class);
+    }
+    
+    /**
+     * 자바 객체인 Task를 Json 형태로 변환
+     * @return Json 형태로 변환된 Task
+     * @throws IOException
+     */
+    private String tasksToJson() throws IOException {
+        //        objectMapper.writeValue(outputStream, tasks);
+        // 새로 Outputstream을 만들어서 리턴해야한다. 이유는 모름 찾아봐야함.
+        OutputStream outputStream = new ByteArrayOutputStream();
+        objectMapper.writeValue(outputStream, tasks);
+
+        return outputStream.toString();
+    }
+
+    /**
+     * Task의 id 자동 증가
+     * @return 1씩 증가된 id
+     */
+    private Long generatedId() {
+        return id += 1;
```

Comment:
return 되는 값이 증가되기 전의 값인가요 아니면 증가된 후의 값인가요? 저라면 명확히 쓸 것 같습니다.
```suggestion
        id += 1;
        return id;
```